src/QuartzNET-DynamoDB/JobStore.cs

```diff
@@ -913,7 +926,37 @@ public void ResumeJob(JobKey jobKey)
 
         public Collection.ISet<string> ResumeJobs(GroupMatcher<JobKey> matcher)
         {
-            throw new NotImplementedException();
+            var resumedGroups = new List<String>();
```

Comment:
It looks like the SQL jobstore executes these in a lock... https://github.com/quartznet/quartznet/blob/d3fe4060e323a5d87225119a540161f55f7b90d5/src/Quartz/Impl/AdoJobStore/JobStoreSupport.cs#L2091